server/src/index.ts

```diff
@@ -42,15 +42,17 @@ app.use("/api-docs", swaggerUi.serve, async (_req: Request, res: Response) => {
   }
 })
 app.get("/", (req: Request, res: Response) => {
-  res.send("Hello World From the Typescript Server!")
+  res.send("UASC backend server is up!")
 })
 
 RegisterRoutes(app)
 
 const port = process.env.PORT || 8000
 
 const _app = app.listen(port, () => {
-  console.log(`Example app listening on port ${port}`)
+  console.log(
+    `UASC backend server listening on port ${port}. http://localhost:${port}`
```

Comment:
this is not actually true - what if it runs on the prod server

Reply:
True, removed that statement!